src/main/java/ui/listpanel/ListPanelCard.java

```diff
@@ -124,8 +130,18 @@ private static Node layoutEvents(Model model, TurboIssue issue,
         result.setSpacing(3);
         VBox.setMargin(result, new Insets(3, 0, 0, 0));
 
-        // Events
+        // Label update events
+        List<TurboIssueEvent> labelUpdateEvents =
+                        events.stream()
+                        .filter(e -> e.isLabelUpdateEvent())
```

Comment:
:/ Ok, could be because of my eclipse's compiler or something. Gradle builds fine. I will change it since this one looks nicer. Also, can help me rebuild it when the CI fail.